public/components/overview/mitre/components/techniques/components/flyout-technique/flyout-technique.tsx

```diff
@@ -204,60 +187,45 @@ export class FlyoutTechnique extends Component {
         title: 'ID',
         description: ( <EuiToolTip
           position="top"
-          content={"Open " + currentTechnique + " details in a new page"}>
-          <EuiLink href={link} external target="_blank">
+          content={"Open " + currentTechnique + " details in Intelligence section"}>
+          <EuiLink onClick={(e) => {this.props.openIntelligence(e,'techniques',currentTechnique);e.stopPropagation()}}>
             {currentTechnique}
           </EuiLink>
         </EuiToolTip>)
       },
       {
-        title: 'Tactic',
-        description: this.getArrayFormatted(
-          techniqueData.phase_name
-        )
-      },
-      {
-        title: 'Platform',
-        description: this.getArrayFormatted(
-          techniqueData.platform_name
-        )
-      },
-      {
-        title: 'Data sources',
-        description: this.getArrayFormatted(
-          techniqueData.x_mitre_data_sources
-        )
+        title: 'Tactics',
+        description: techniqueData.tacticsObj
+        ? techniqueData.tacticsObj.map((tactic) => {
+            return (
+              <>
+                <EuiToolTip
+                  position="top"
+                  content={
+                    "Open " + tactic.name + " details in a Intelligence section"
```

Comment:
spell
`details in the Intelligence section`

Reply:
Fixed and replaced by template string